test/unit/notify/send-emails.test.js

```diff
@@ -79,33 +79,18 @@ describe('send emails', () => {
     expect(sendEmail.mock.calls[1]).toEqual([mockContext, mockMessage, defaultEmailAddresses[1], defaultReference])
   })
 
-  test('should throw error when sendEmail rejects', async () => {
+  test('should resolve in sendEmails, thrown error when sendEmail rejects', async () => {
     sendEmail.mockRejectedValue(new Error('Oh dear'))
 
-    const wrapper = async () => {
-      await sendEmails(mockContext, mockMessage)
-    }
-
-    expect(wrapper).rejects.toThrow()
-  })
-
-  test('should throw Error when sendEmail rejects', async () => {
-    sendEmail.mockRejectedValue(new Error('Oh dear'))
-
-    const wrapper = async () => {
-      await sendEmails(mockContext, mockMessage)
-    }
-
-    expect(wrapper).rejects.toThrow(Error)
+    expect(sendEmails(mockContext, mockMessage)).resolves.not.toThrow()
   })
 
-  test('should throw "Oh dear" Error when sendEmail rejects', async () => {
-    sendEmail.mockRejectedValue(new Error('Oh dear'))
-
-    const wrapper = async () => {
-      await sendEmails(mockContext, mockMessage)
-    }
-
-    expect(wrapper).rejects.toThrowError(/^Oh dear/)
+  test('confirm error thrown by sendEmail is caught and logged', async () => {
+    const errorMock = new Error('Oh dear')
+    sendEmail.mockRejectedValue(errorMock)
+    
+    await sendEmails(mockContext, mockMessage)
+    const logSpy = jest.spyOn(mockContext, 'log')
+    expect(logSpy).toHaveBeenCalledWith(errorMock)
```

Comment:
Consider rearranging as such:

const errorMock = new Error('Oh dear')
    const logSpy = jest.spyOn(mockContext, 'log')
    sendEmail.mockRejectedValue(errorMock)

    await sendEmails(mockContext, mockMessage)
    
    expect(logSpy).toHaveBeenCalledWith(errorMock)
})


This way it represents the 

- setup needed (mockRejectedValue and logspy),
- the actual execution (sendEmails) and 
- the comparison operator(s) (expect(...))